src/containers/shared/components/CopyToClipboard.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import copy from '../images/copy.png'
+
+interface Props {
+  text: string
+  className?: string
+}
+
+const CopyToClipboard = ({ text, className }: Props) => (
+  <input
+    className={className}
+    type="image"
+    src={copy}
+    alt="Copy"
+    onClick={() => navigator.clipboard.writeText(text)}
+  />
+)
+CopyToClipboard.propTypes = {
```

Comment:
There is need to specify propTypes or defaultProps for TSX components.

Reply:
could you elaborate please? i thought i included them? or do you mean i dont need them?

Reply:
Opps sorry. I meant there is no need to specify proptypes